src/content/guides/creating-clusters-via-crs-on-aws/index.md

```diff
@@ -223,6 +223,27 @@ As a result of rendering the CRs ([sample](/reference/kubectl-gs/template-cluste
 The resources can then be created by applying the manifest files to the Control Plane, e.g. `kubectl create -f <cluster manifest file>.yaml`.
 Of course, that requires the user to be authorized towards Kubernetes Control Plane API.
 
+## How to configure OIDC authentication at cluster creation using Cluster API
```

Comment:
That's pretty long for an intermediate headline, and it duplicates some context info. I would suggest:

```markdown
## Configuring OIDC authentication
```